bin/compile

```diff
@@ -58,3 +58,5 @@ $builder->setStub(
 );
 
 chmod(__DIR__ . '/couscous.phar', 0755);
+
+exec('sha1sum bin/couscous.phar > bin/couscous.version');
```

Comment:
Could we use something more widespread? I don't have that installed, I have `md5` though. We can also do it with PHP (`php -r "..."`) if needed.